src/document/docUtil.ts

```diff
@@ -1,6 +1,6 @@
 type Selection = {
-    canCreate: boolean;
     containerRect: DOMRect;
+    error: boolean;
```

Comment:
I guess I was thinking this could be a string that describes the error (in case we ever needed to add more) but that may be overcomplicating it. If it's just a boolean, could we name it `hasError`? And does it have to be defined or can we keep it optional?